src/code-templates/libraries/logger/test/index.test.ts

```diff
@@ -7,77 +7,71 @@ beforeEach(() => {
 });
 
 describe('logger', () => {
-  test('When no explicit configuration is set, info logs are written', async () => {
-    // Arrange
-    const stdoutStub = sinon.stub(process.stdout, 'write');
+  let stdoutStub: sinon.SinonStubbedMember<typeof process['stdout']['write']>;
 
+  beforeEach(() => {
+    // Must stub the process.stdout before we configure the logger
+    // as it would fail when the underlying logger is pino, and we run in WebStorm
+    // See more here - practicajs/practica#225
+    stdoutStub = sinon.stub(process.stdout, 'write');
+  });
+
+  test('When no explicit configuration is set, info logs are written', async () => {
     // Act
     logger.info('This is an info message');
 
     // Assert
-    expect({ stdCallCount: stdoutStub.callCount }).toMatchObject({
-      stdCallCount: 1,
-    });
-    const lastStdoutCall = JSON.parse(stdoutStub.lastCall?.firstArg);
```

Comment:
I did this intentionally so the test result will look like:

expected {stdCallCount:1) but got {stdCallCount:0) and the tests result will not look like:

expected 1 but got 0

Reply:
This is why I prefer using jest spies as there matchers that much more helpful than doing this as it gives you more context of the actual argument.

Maybe we should use sinon matchers for jest?

Reply:
Yes!

p.s. Jest mocking is a world full with magics, this is why I avoid it all together (even the good parts)

Reply:
reverted back to use stdCallCount